src/app/design/frontend/base/default/layout/omise/gateway.xml

```diff
@@ -12,4 +12,19 @@
             </action>
         </reference>
     </checkout_onepage_index>
+    <omise_gateway_checkout_tesco_index translate="label">
+        <label>Omise Checkout Success</label>
+        <reference name="head">
+            <action method="addItem">
+                <type>skin_css</type>
+                <name>omise/gateway/css/style.css</name>
```

Comment:
I can't remember exactly how CSS inclusion works - I assume there's no danger that this might result in `style.css` being added to the page twice?

Reply:
This includes CSS once.

Reply:
What I meant was that this CSS file is already used in other payment methods. I was wondering if the way in which it is included for those methods could possibly result in a double inclusion along with this one, or is Magento smart enough to realise that two or more components are sharing this resource, and only include it once

Reply:
Basically, the section which I've added in layout file is telling Magento that, use CSS in Tesco's Checkout success page. Other section in the same layout file tells Magento to add CSS only in Checkout page. So there is no danger of double inclusion.

Reply:
OK, makes sense